/**/controller_manager:
  ros__parameters:
    update_rate: 125  # Hz

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

    psyonic_left_hand:
      type: joint_trajectory_controller/JointTrajectoryController


/**/psyonic_left_hand:
  ros__parameters:
    allow_partial_joints_goal: True
    joints:
      - Left_Hand_index_q1
      - Left_Hand_middle_q1
      - Left_Hand_ring_q1
      - Left_Hand_pinky_q1
      - Left_Hand_thumb_q2
      - Left_Hand_thumb_q1
 
    command_interfaces:
      - position
    state_interfaces:
      - position
      - velocity
      
    # Optional: Add trajectory execution parameters
    constraints:
      stopped_velocity_tolerance: 0.01
      goal_time_tolerance: 0.5
      
    # Action server configuration
    action_server:
      allow_partial_joints_goal: true